docs/kubernetes/deploy.md

```diff
@@ -67,6 +67,8 @@ kubernetes-dashboard is running at https://contoso-apple-59769a59.westus2.clouda
 To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
 ```
 
+Administrative note: By default, the directory where acs-engine stores cluster configuration (`_output/contoso-apple` above) won't be overwritten as a result of subsequent attempts to deploy a cluster using the same `--dns-prefix`) To re-use the same resource group name repeatedly, include the `--force-overwrite` command line option with your `acs-engine deploy` command. On a related note, include an `--auto-suffix` option to append a randomly generated suffix to the dns-prefix, if your workflow requires a common prefix across multiple cluster deployments. Using the `--auto-suffix` pattern will guarantee that each deployment's configuration artifacts will be stored locally under a discrete `_output/<resource-group-name>/` directory.
```

Comment:
I would just make it clear that auto-suffix `appends a compressed timestamp to the dnsPrefix to ensure a unique cluster name`, as opposed to only storing the output in a unique directory. Otherwise looks good.